docs/ProjectSpec.md

```diff
@@ -219,6 +219,7 @@ A dependency can be one of a 3 types:
 These only applied to `target` and `framework` dependencies.
 
 - ⚪️ **embed**: `Bool` - Whether to embed the dependency. Defaults to true for application target and false for non application targets.
+- ⚪️ **link**: `Bool` - Whether to link the dependency. Defaults to true for but only static library and dynamic frameworks are linked.
```

Comment:
`for but`
And maybe mention that this is only applicable for `target` dependency types

Reply:
Updated!